maintainers/maintainer-list.nix

```diff
@@ -6486,6 +6486,12 @@
     githubId = 131856;
     name = "Arnout Engelen";
   };
+  RaghavSood = {
```

Comment:
please separate this into another commit

Reply:
Done